.github/workflows/beam_PostCommit_Java_ValidatesRunner_Direct.yml

```diff
@@ -75,9 +75,7 @@ jobs:
           distribution: 'zulu'
           java-version: '8'
       - name: run validatesRunner script
-        uses: ./.github/actions/gradle-command-self-hosted-action
-        with:
-          gradle-command: :runners:direct-java:validatesRunner
+        run: ./gradlew :runners:direct-java:validatesRunner
```

Comment:
These changes from using composite action to a command line directly. What was the problem for gradle-command-self-hosted-action here, should we focus on fix the action instead?

If it is some default options in `gradle-command-self-hosted-action`, should we have an option to control those options?

Reply:
Yes, I noticed that the Jenkins job doesn't use the default arguments like `-Pdocker-pull-licenses`. As an alternative way we can specify the `default-arguments` as empty:
```
      - name: run validatesRunner script
        uses: ./.github/actions/gradle-command-self-hosted-action
        with:
          gradle-command: :runners:direct-java:validatesRunner
          default-arguments: ''
```
But there is a `max-workers` argument that should also be handled somehow. So, I found running it as a command line easier than changing action and adding additional validation.